packages/repository/src/repositories/kv.repository.ts

```diff
@@ -6,7 +6,28 @@
 import {Repository} from './repository';
 import {Options, DataObject} from '../common-types';
 import {Model} from '../model';
-import {Filter} from '../query';
+
+/**
+ * Filter for keys
+ */
+export type KVFilter = {
+  /**
+   * Glob string to use to filter returned keys (i.e. `userid.*`). All
+   * connectors are required to support `*` and `?`. They may also support
+   * additional special characters that are specific to the backing database.
+   */
+  match: string;
+};
+
+/**
+ * Polyfill for AsyncIterator before es.next is ready
+ */
+// tslint:disable:no-any
+export interface AsyncIterator<T> {
```

Comment:
Do we still need this poly-fill after you enabled `esnext.asynciterable` in tsconfig?

Reply:
Removed